tests/code/test_AdminNewEvent.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from app.logic.adminNewEvent import manageNewEventData, createEvent
+
+@pytest.mark.integration
+def test_manageNewEventData():
+
+    # test that there is a return
+    assert manageNewEventData({})
+
+    # tets for return type
+    assert type(manageNewEventData({}))== type({})
+
+    # test for no keys
+    eventData = {}
+    newData = manageNewEventData(eventData)
+    assert newData['eventRequiredForProgram'] == False
+    assert newData['eventRSVP'] == False
+    assert newData['eventServiceHours'] == False
+    assert newData['eventIsTraining'] == False
+
+    #test for one missing key
+    eventData = {'eventRequiredForProgram':'on','eventRSVP':'', 'eventServiceHours':True }
+    newData = manageNewEventData(eventData)
+
+    assert newData['eventIsTraining'] == False
+    with pytest.raises(AssertionError):

```

Comment:
I'm not quite sure what the negative assertion is here. If you just want to see if the data is still there, check that it has the right value

Reply:
The value of newData['eventIsTraining'] is False

Reply:
What I mean is, why are you checking to see if an assert raises an exception rather than just asserting the value you expect directly?

Reply:
We have removed the exception and use assert to test.